kie-server-parent/kie-server-controller/kie-server-controller-impl/pom.xml

```diff
@@ -60,6 +60,14 @@
       <groupId>com.thoughtworks.xstream</groupId>
       <artifactId>xstream</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.kie.server</groupId>
+      <artifactId>kie-server-services-common</artifactId>
```

Comment:
Is this dependency needed? From code changes it looks like kie-server-common should be enough.

Reply:
You are right. Fixed